src/gameplay/player/movement.rs

```diff
@@ -11,16 +11,19 @@ use crate::fixed_update_inspection::did_fixed_update_happen;
 use super::default_input::{Jump, Move};
 
 use super::PLAYER_FLOAT_HEIGHT;
+use super::noclip::is_noclipping;
 use super::{Player, camera::PlayerCamera};
 
 pub(super) fn plugin(app: &mut App) {
     app.add_systems(
         FixedUpdate,
-        apply_movement.in_set(TnuaUserControlsSystemSet),
+        apply_movement
+            .in_set(TnuaUserControlsSystemSet)
+            .run_if(not(is_noclipping)),
```

Comment:
I think we don't need this `run_if`, since the input is being cleared anyways in noclip, no?

Reply:
We actually need it! Otherwise the character does move while in noclip mode.